### DISCUSSION TITLE
Add html-indent for template plugin.

### DISCUSSION THREAD

**Discussion by tanoargie** [angular-eslint/angular-eslint#564] [New Rule Proposals]
Hi all,
First, let me apologize if this is not the place to ask for this.
I have started my journey in the Angular world pretty recently so I might be wrong, but as I understand, right now there is no rule in the package [eslint-plugin-template](https://github.com/angular-eslint/angular-eslint/tree/master/packages/eslint-plugin-template) to avoid the following:

<img width="1358" alt="Screen Shot 2021-06-29 at 17 06 05" src="https://user-images.githubusercontent.com/16160759/123864427-21c9cc80-d901-11eb-99ca-dab110180d6a.png">

For example the **eslint-plugin-vue** plugin has a rule with the following options (you can check it [here](https://eslint.vuejs.org/rules/html-indent.html)):

```json
{
  "vue/html-indent": ["error", 2, {
    "attribute": 1,
    "baseIndent": 1,
    "closeBracket": 0,
    "alignAttributesVertically": true,
    "ignores": []
  }]
}
```

Is this something we could add to the template plugin? Is there a simpler option I am not seeing? If there is some agreement I will make an issue and try to implement it.
Thanks in advance!

**Comment by rafaelss95** [comment]
Hey @francoserio, we recommend using formatters to indent as they can do this much better than a linter.

**Comment by JamesHenry** [comment]
Hi @francoserio I took the opportunity to formally write up our view on using a linter to enforce code formatting concerns here: 
https://github.com/angular-eslint/angular-eslint/blob/master/docs/FORMATTING_RULES.md

**Comment by tanoargie** [comment]
Ok, thank you very much @rafaelss95 for the quick response! Also, thanks @JamesHenry for having written the position for future concerns about this!
